VHbbAnalysis/Heppy/test/vhbb.py

```diff
@@ -389,8 +396,8 @@ def ele_mvaEleID_Trig_preselection(ele) :
     verbose = False,
     class_object = TriggerBitAnalyzer,
     triggerBits = triggerTable,  #default is MC, use the triggerTableData in -data.py files
-    processName = 'HLT',
-    fallbackProcessName = 'HLT2',
+   processName = 'HLT',
+#   outprefix = 'HLT'
```

Comment:
this is undoing the fix from gregor that would make useless the fix_reHLT function below... can you please restore the automatic fix with fallbackProcessName?